frontstage/controllers/party_controller.py

```diff
@@ -510,3 +511,39 @@ def notify_party_and_respondent_account_locked(respondent_id, email_address, sta
         raise ApiError(logger, response)
 
     bound_logger.info('Successfully notified respondent and party service that account is locked')
+
+
+def get_list_of_business_for_party(party_id):
+    bound_logger = logger.bind(party_id=party_id)
+    bound_logger.info('Getting enrolment data for the party')
+    enrolment_data = list(get_respondent_enrolments(party_id))
+    business_ids = set()
+    for enrolment in enrolment_data:
+        business_ids.add(enrolment['business_id'])
+    bound_logger.info('Getting businesses against business ids')
+    return get_business_by_business_id(list(business_ids))
+
+
+def get_business_by_business_id(business_ids):
```

Comment:
There is a function that does this already (albeit with a slightly different url) `get_party_by_business_id` is this new one needed?

Reply:
done!